conformance/base/manifests.yaml

```diff
@@ -91,9 +91,15 @@ spec:
   selector:
     app: infra-backend-v1
   ports:
-  - protocol: TCP
+  - name: first-port
```

Comment:
So I don't want to put the protocol name here. It might inadvertedly inform the implementation what the backend protocol is.

eg. istio if you put http2 it will assume h2c

Reply:
oh, well then can be disregarded